fdbrpc/sim2.actor.cpp

```diff
@@ -1727,6 +1739,202 @@ class Sim2 final : public ISimulator, public INetworkConnections {
 	int yield_limit;  // how many more times yield may return false before next returning true
 };
 
+class UDPSimSocket : public IUDPSocket, ReferenceCounted<UDPSimSocket> {
+	using Packet = std::shared_ptr<std::vector<uint8_t>>;
+	UID id;
+	ISimulator::ProcessInfo* process;
+	Optional<NetworkAddress> peerAddress;
+	Optional<ISimulator::ProcessInfo*> peerProcess;
+	Optional<Reference<UDPSimSocket>> peerSocket;
+	ActorCollection actors;
+	Promise<Void> closed;
+	std::deque<std::pair<NetworkAddress, Packet>> recvBuffer;
+	AsyncVar<int64_t> writtenPackets;
+	NetworkAddress _localAddress;
```

Comment:
Why is this field named with a leading underscore but none of the other fields are?

Reply:
because `localAddress` is a method within this class. It's easier to give the field a different name